app/assets/javascripts/client/home/testimonal.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .directive('testimonial', testimonial);
+
+  function testimonial() {
+    return {
+      restrict: 'E',
+      transclude: true,
+      replace: true,
+      scope: {
+        image: '@',
+        backgroundStyle: '@',
+        learnMoreLink: '@',
+        quote: '@',
+        author: '@',
+        subtext: '@',
+        toolName: '@',
+        alternate: '@'
+      },
+      templateUrl: 'client/home/testimonial_template.html'
```

Comment:
I've noticed that this version of Angular will get very mixed up when compiling the assets if the name of the directive and the name of the template are similar enough, since it all gets compiled down to HTML.  I can experiment but I don't think that this is likely to change.